app/renderer/ui/preference-view/components/button.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+const props = defineProps({
+  title: {
+    type: String,
+    required: true,
+  },
+  info: {
+    type: String,
+    required: true,
+  },
+});
+
+const emits = defineEmits(["event:change"]);
+</script>
+
+<template>
+  <div class="flex justify-between">
```

Comment:
could you please provide a screenshot of this component?

I wish to check the UI looking. thanks.

Reply:
![image](https://github.com/user-attachments/assets/d7ee159c-39a7-4a8c-98d8-513af1b67760)

Reply:
UI looks ok.